### ISSUE TITLE
Backup seems to soft fail for games on the same drive as the backup destination

### ISSUE THREAD

**Issue by ThaChillera** [mtkennerly/ludusavi#107]
I use ludusavi to grab all important game files and put them in the same folder, before using a different tool to backup to a remote location.

However, it seems that some games aren't getting proper backups. Games installed on the main drive, where steam is installed & the backup is done to, just get an empty folder for backup. when reviewing the logs or viewing the list op found files in the GUI, all files are listed as expected.

**Comment by mtkennerly** [comment]
Hmm, having everything on the same drive is how I normally use/test it, so I'm not exactly sure what's going wrong here.

* Is this something that used to work in 0.10.0 and is having an issue now in the preview build from #104 (0.10.0-post.29+9d2547a)?
* Could you please share a screenshot of a working game (file list in Ludusavi + populated backup folder) and a non-working game (file list in Ludusavi + empty backup folder)?
* Could you share your config file? (Linux: `$XDG_CONFIG_HOME/ludusavi` or `~/.config/ludusavi`)

**Comment by ThaChillera** [comment]
While creating the comparison list for you, I realised that steam's default install location is in .local. So all backups were in drive_0/home/{username}/.local, and my file browser of course hid the dot folder, making it look like nothing was backed up!

tl;dr no issues! thanks for the help, sorry for wasting your time :)

**Comment by mtkennerly** [comment]
Phew 😅 That's good to hear. No worries!
